tests/Directory.Packages.props

```
@@ -1,16 +1,16 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
   <PropertyGroup>
-    <TestcontainersPackageVersion>4.6.0</TestcontainersPackageVersion>
+    <TestcontainersPackageVersion>4.7.0</TestcontainersPackageVersion>
   </PropertyGroup>
   <ItemGroup>
     <!-- unit test dependencies -->
     <PackageVersion Include="bUnit" Version="1.36.0" /> <!-- Can't update passed to 1.37.x versions as those lift up LTS versions when targeting net8 -->
-    <PackageVersion Include="JsonSchema.Net" Version="7.3.4" />
+    <PackageVersion Include="JsonSchema.Net" Version="7.4.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing" Version="1.1.2" />
     <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorVersion)" />
     <PackageVersion Include="Microsoft.DotNet.XUnitV3Extensions" Version="$(MicrosoftDotNetXUnitV3ExtensionsVersion)" />
-    <PackageVersion Include="Microsoft.Playwright" Version="1.52.0" />
+    <PackageVersion Include="Microsoft.Playwright" Version="1.54.0" />
```

```suggestion
    <PackageVersion Include="Microsoft.Playwright" Version="1.52.0" />
```

This needs to be reverted. See https://github.com/dotnet/aspire/pull/10429